docs/get-started/quickstarts/detect-people.md

```diff
@@ -42,7 +42,10 @@ Click the name of a machine to go to that machine's page, where you'll find a va
 {{% /expand%}}
 {{%expand "Step 2: Install viam-server" %}}
 
-Follow the {{< glossary_tooltip term_id="setup" text="setup instructions" >}} that appear on your new machine's **CONFIGURE** page to install `viam-server` on your computer and connect it to the Viam app.
+Follow the {{< glossary_tooltip term_id="setup" text="setup instructions" >}} that appear on your new machine's **CONFIGURE** page to install `viam-server` on your computer, start running it and connect it to the Viam app.
+Select the OS that's appropriate for your computer.
+For example, **Mac** for a MacBook.
+Select **RDK** as the RDK type.
```

Comment:
Do we think that's necessary? I had just removed a lot of these because it feels like we're stating the obvious...

Reply:
I don't know if people will know the RDK thing if they're brand new users. Can remove the "for example" maybe?